src/philips_airctrl/discovery.py

```
@@ -0,0 +1,154 @@
+                    for addr_info in addrs[netifaces.AF_INET]:
+                        ip = addr_info.get("addr")
+                        netmask = addr_info.get("netmask")
+
+                        if ip and netmask and not ip.startswith("127."):
+                            try:
+                                network = ipaddress.IPv4Network(f"{ip}/{netmask}", strict=False)
+                                networks.append(str(network))
+                            except (ipaddress.AddressValueError, ValueError):
+                                continue
+
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)